test/e2e/kubevirt.go

```diff
@@ -972,14 +1002,32 @@ passwd:
 			waitForPodsCondition(httpServerTestPods, conditionFn)
 			httpServerTestPods = updatePods(httpServerTestPods)
 		}
+
+		removeVMImages = func(node string) {
+			By("Removing unused images in node " + node)
+			output, err := runCommand(containerRuntime, "exec", node,
+				"crictl", "images", "-o", "json")
+			Expect(err).NotTo(HaveOccurred())
+
+			imageID, err := images.ImageIDByImageURL(strings.Split(kubevirt.FedoraContainerDiskImage, ":")[0], output)
```

Comment:
Can we parameterize the image so we have 
```golang
removeVMImages(node, image string)
```

This way we can use this for the fedoracores image too.

Reply:
ACK, DONE